src/plugins/content_management/tsconfig.json

```diff
@@ -14,6 +14,7 @@
     "@kbn/object-versioning",
     "@kbn/core-saved-objects-api-server-mocks",
     "@kbn/core-saved-objects-api-server",
+    "@kbn/saved-objects-finder-plugin",
```

Comment:
Unfortunately this causes problems. `content_management` dependencies need to be minimized.

`@kbn/saved-objects-finder-plugin` depends upon `@kbn/saved-objects-management-plugin` depends upon the `data` and `data-view` plugins which means its a circular dependency if they then try to implement the CM api. 

I guess there are other places in that dependency chain where changes could be made but this seems like a simple place.